src/components/Footer/Footer.js

```diff
@@ -1,7 +1,32 @@
 import React from 'react';
+import styled from 'styled-components';
+
+const FooterStyle = styled.div`
+  .Footer {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100px;
+    border: 1px gray solid;
+    color: rgb(134, 133, 133);
+  }
+`;
```

Comment:
```suggestion
const FooterStyle = styled.footer`
    display: flex;
    align-items: center;
    justify-content: center;
    height: 100px;
    border: 1px gray solid;
    color: rgb(134, 133, 133);
`;
```
1. 시멘틱 태그를 사용해주세요! footer 태그~
2. styled-component를 정의한 후 내부에서 클래스명을 사용하는 것이 아니라 바로 스타일링을 적용해주시면 됩니다!

Reply:
추가로 styled-component 의 위치를 최 하단으로 옮겨주세요! (가독성을 위함입니다!)